### ISSUE TITLE
Throw error when none of the root partitions are found

### ISSUE THREAD

**Issue by MartijnBraam** [postmarketOS/pmbootstrap#171]
The find_root_partition (https://github.com/postmarketOS/pmbootstrap/blob/a502b5047580f365947587ea4a26cfc688b87903/aports/postmarketos-mkinitfs/init_functions.sh#L56) function doesn't handle a device without a flashed system partition at all. It should at least give a clear error message.

The current output for @PureTryOut is:

```
### postmarketOS initramfs ###
Device  doesn't exist or access denied.
---
WARNING: usb shell is active on 172.16.42.1:24.
This is a security hole! Only use it for debugging, and
uninstall the usb-shell hook afterwards!
```

The Device doesn't exist is caused by find_root_partition not returning anything.

**Comment by PabloCastellano** [comment]
It would be helpful to show the output of `blkid` after the error message. For example:

```
/dev/mmcblk0p25: UUID="57f8f4bc-abf4-655f-bf67-946fc0f9f25b" TYPE="ext4"
/dev/mmcblk0p24: UUID="57f8f4bc-abf4-655f-bf67-946fc0f9f25b" TYPE="ext4"
/dev/mmcblk0p23: LABEL="system" UUID="da594c53-9beb-f85c-85c5-cedf76546f7a" TYPE="ext4"
/dev/mmcblk0p18: UUID="57f8f4bc-abf4-655f-bf67-946fc0f9f25b" TYPE="ext4"
```

**Comment by drebrez** [comment]
And maybe we can use the `busybox findfs` applet (already included in the actual busybox we use) to find the root partition.

```
/ # /sbin/findfs --help
BusyBox v1.27.0 (2017-07-07 08:16:37 GMT) multi-call binary.

Usage: findfs LABEL=label or UUID=uuid

Find a filesystem device based on a label or UUID
```

**Comment by ollieparanoid** [comment]
@drebrez [had the good idea](https://github.com/postmarketOS/pmbootstrap/pull/206#issuecomment-317199194) of showing another splash screen, when the system partition was not found. Now that we can generate splash screens on the fly thanks to efforts by @MartijnBraam, it should be easy to implement this.

I suggest doing both, showing the splash screen and some useful logging output - a message, and the output from `blkid`.

**Comment by ollieparanoid** [comment]
Implemented by @drebrez in #257 👍
